manifests/charts/base/templates/services.yaml

```diff
@@ -24,8 +10,6 @@ spec:
     name: tcp-istiod
   clusterIP: None
   {{- end }}
----
```

Comment:
the yamldoc start (I believe) is still needed. Did you manually generate this manifest and ensure the service that follows is created?